<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Page3</class>
 <widget class="QWidget" name="Page3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1191</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="BackButton">
   <property name="geometry">
    <rect>
     <x>1081</x>
     <y>596</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button1_Insert">
   <property name="geometry">
    <rect>
     <x>981</x>
     <y>596</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>插入排序</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextStep">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>590</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>下一步</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button5_Quick">
   <property name="geometry">
    <rect>
     <x>1081</x>
     <y>561</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>快速排序</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button2_Binary">
   <property name="geometry">
    <rect>
     <x>881</x>
     <y>596</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>二分排序</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button6_Selection">
   <property name="geometry">
    <rect>
     <x>981</x>
     <y>561</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>选择排序</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button7_Heap">
   <property name="geometry">
    <rect>
     <x>881</x>
     <y>561</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>heap排序</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button3_Shell">
   <property name="geometry">
    <rect>
     <x>781</x>
     <y>596</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>希尔排序</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button8_Merge">
   <property name="geometry">
    <rect>
     <x>781</x>
     <y>561</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>归并排序</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button9_Radix">
   <property name="geometry">
    <rect>
     <x>681</x>
     <y>561</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>基数排序</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button4_Bubble">
   <property name="geometry">
    <rect>
     <x>681</x>
     <y>596</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>冒泡排序</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonShuffle">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>540</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>打乱数组</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
